config.md

```diff
@@ -245,7 +245,7 @@ Presently there are `Prestart`, `Poststart` and `Poststop`.
 * [`Poststop`](#poststop) is a list of hooks to be run after the container process exits
 
 Hooks allow one to run code before/after various lifecycle events of the container.
-Hooks MUST be called in the listed order.
+Hooks MUST be called in the listed order if supported by the platform.
```

Comment:
This is fine only if the bundle were to support different platforms else one could just not add hooks to the config at all.